tests/test_skills.py

```diff
@@ -303,6 +353,10 @@ def test_linear_skillset():
     )
 
     agent.learn(learning_iterations=2)
+    assert agent.skills[
+               "skill_0"].instructions == '"Given a category and its context, provide a list of items that belong to this category. The context will help in generating a more accurate list. Do not provide definitions or explanations, just list the items."'  # noqa: E501
+    # TODO: not learned with 2 iterations, need to increase learning_iterations
+    assert agent.skills["skill_1"].instructions == '...'
```

Comment:
@pakelley you were running into a bug where `...` was generated as the openai response due to missing a field in the runtime or something, right? has that happened here?

Reply:
here the effect is due to the "live" test calls that don't reach second skill `skill_1` as `skill_0` remains unoptimized during 2 iterations